pkgs/build-support/mkfont.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, stdenvNoCC }:
+
+args:
+
+let
+  noUnpackFonts = lib.hasAttr "noUnpackFonts" args && args.noUnpackFonts;
+  hasMultipleSources = lib.hasAttr "srcs" args;
+in
+stdenvNoCC.mkDerivation ({
+  dontPatch = true;
```

Comment:
Wouldn’t it make sense to instead provide defaults that can be overridden here?

Reply:
Everything can be overridden via the `// args` below, and I do that in a couple of derivations (e.g. `dontBuild = false;` and provide a build phase on some). Did you have another mechanism for that in mind?

Reply:
You're correct, thanks for the clarification.

Reply:
Right, but this is inconsistent with other builders so people might be confusing when e.g. patches do not work. It fits more in with trivial builders. Perhaps it should be named `installFontFiles` and be part of trivial builders.